admin/Openedx_Woocommerce_Plugin_Admin.php

```diff
@@ -105,85 +110,138 @@ public function enqueue_styles() {
-         * between the defined hooks and the functions defined in this
-         * class.
-         */
-		
+		 *
+		 * The Openedx_Woocommerce_Plugin_Loader will then create the relationship
+		 * between the defined hooks and the functions defined in this
+		 * class.
+		 */
 	}
 
-     /**
-     * Register Enrollment Request custom post type 
-     *
-     * @since    1.0.0
-     */
-    public function register_enrollment_custom_post_type() {
+	/**
+	 * Register Enrollment Request custom post type 
+	 *
+	 * @since    1.0.0
+	 */
+	public function register_enrollment_custom_post_type()
+	{
 
 		$this->openedx_enrollment->register_enrollment_custom_post_type();
-
 	}
 
 	/**
 	 * Render Enrollment Request info form
 	 *
 	 * @since    1.0.0
 	 */
-	public function render_enrollment_info_form($post) {
+	public function render_enrollment_info_form($post)
+	{
 
 		$this->openedx_enrollment_info_form = new Openedx_Woocommerce_Plugin_Enrollment_Info_Form($post);
+	}
+
+	/**
+	 * Wrapper function to register a new post type
+	 *
+	 * @param  string $post_type   Post type name.
+	 * @param  string $plural      Post type item plural name.
+	 * @param  string $single      Post type item single name.
+	 * @param  string $description Description of post type.
+	 * @return object              Post type class object
+	 */
+	public function register_post_type($post_type = '', $plural = '', $single = '', $description = '', $options = array())
+	{
+
+		if (!$post_type || !$plural || !$single) {
+			return;
+		}
+
+		$post_type = $this->createPostType($post_type, $plural, $single, $description, $options);
+
+		return $post_type;
+	}
+
+	/**
+	 * Create a new instance of the Openedx_Woocommerce_Plugin_Post_Type class and register a new post type.
+	 *
+	 * @param  string $post_type   Post type name.
+	 * @param  string $plural      Post type item plural name.
+	 * @param  string $single      Post type item single name.
+	 * @param  string $description Description of the post type.
+	 * @param  array  $options     Additional options for the post type.
+	 * @return object              Post type class object.
+	 */
+	public function createPostType($post_type = '', $plural = '', $single = '', $description = '', $options = array())
+	{
+
+		return new Openedx_Woocommerce_Plugin_Post_Type($post_type, $plural, $single, $description, $options);
+	}
 
+	/**
+	 * Register course ID and mode fields for product
+	 *
+	 * @since    1.1.1
+	 */
+	function add_custom_product_fields()
+	{
+		global $post;
+
+		echo '<div class="options_group">';
+
+		echo '<p class="form-field">' . __('Only use these fields if the product is an Open edX course.', 'woocommerce') . '</p>';
+
+		woocommerce_wp_text_input(array(
+			'id'          => '_course_id',
+			'label'       => __('Course ID', 'woocommerce'),
+			'placeholder' => '',
+			'desc_tip'    => 'true',
+			'description' => __('Enter the course ID for the product.', 'woocommerce')
```

Comment:
```suggestion
			'description' => __('Ex: course-v1:edX+DemoX+Demo_Course.<br> You can find the Open edX Course ID in the URL of your course in your LMS.', 'woocommerce')
```
I don't know if the correct form to add a line break in that description is "br"; if not, please correct me. If we can't add a line break is okay too.

Reply:
Yes, is correct to put a <br> to add a line break but it's better to add two <br> because if we add only one is not going to change anything.

Here is the text with and without a single line break:
![image](https://github.com/eduNEXT/openedx-woocommerce-plugin/assets/52968528/5739332c-37fa-42b2-9883-dc69473e1076)

This is with two <br>:
![image](https://github.com/eduNEXT/openedx-woocommerce-plugin/assets/52968528/058fcb62-ad8c-40a2-96b6-151c6a50e47c)